### PR TITLE
[tests-only]Run GUI tests parallelly

### PR DESCRIPTION
**Pull request by Talank**
fixes: https://github.com/owncloud/client/issues/9567

Screenshot from 2022-06-16 17-06-51

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12134/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12134/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12134/guiReportUpload/serverlog.log

</details>

File: .drone.star [hunk 1]
```diff
@@ -36,6 +36,20 @@ dir = {
...
+    "tst_addAccount",
+    "tst_syncing",
+    "tst_checkAlltabs",
+    "tst_vfs",
+    "tst_deletFilesFolders",
+    "tst_editFiles",
+    "tst_loginLogout",
+    "tst_removeAccountConnection",
+]
+
```
Comment (saw-jan) [inline .drone.star:52]: Maybe we can use `config` like in other repos:
```
config = {
  gui_tests_1: [
    "tst_sharing",
  ], 
  gui_tests_1: [
    "tst_addAccount",
    ...
  ]
}
```

and refactor `gui_tests` to build pipelines as per the config

**Review by sushmita56 — COMMENTED** [review]
LGTM

**Comment by saw-jan** [comment]
we will be working on this later.

**Comment by saw-jan** [comment]
this is not of higher priority.
Closing here!

**Pull request closed without merging**